docs/core/tutorial/01-etl-before-prefect.md

```diff
@@ -12,7 +12,10 @@ Grab the tutorial code:
 
 ```
 git clone --depth 1 https://github.com/PrefectHQ/prefect.git
-cd prefect/examples/tutorial
+cd prefect
+python3 -m venv env  
+source env/bin/activate   
+cd examples/tutorial
```

Comment:
That seems fine. As a tutorial it seemed important to model good practices rather than encouraging people to install stuff system wide. I suggested venv because it's part of the python standard library so should be available to anyone